app/viewmodels/event2/checkAnswers/DatePaidSummary.scala

```diff
@@ -18,39 +18,34 @@ package viewmodels.event2.checkAnswers
 
 import models.common.MembersDetails
 import models.enumeration.EventType.Event2
-
-import java.time.format.DateTimeFormatter
 import models.{Index, UserAnswers}
 import pages.common.MembersDetailsPage
-import pages.{CheckAnswersPage, Waypoints}
 import pages.event2.DatePaidPage
+import pages.{CheckAnswersPage, Waypoints}
 import play.api.i18n.Messages
 import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
 import utils.Event2MemberPageNumbers
 import viewmodels.Message
 import viewmodels.govuk.summarylist._
 import viewmodels.implicits._
 
-object DatePaidSummary  {
+import java.time.format.DateTimeFormatter
+
+object DatePaidSummary {
 
   def row(answers: UserAnswers, waypoints: Waypoints, sourcePage: CheckAnswersPage, index: Index)
          (implicit messages: Messages): Option[SummaryListRow] =
     answers.get(DatePaidPage(index, Event2)).map {
       answer =>
-        //TODO: remove duplicate code, find more effective method
-        val beneficiaryFullNameOpt = answers.get(MembersDetailsPage(Event2, index, Event2MemberPageNumbers.SECOND_PAGE_BENEFICIARY)).map {
-          g => MembersDetails(g.firstName, g.lastName, g.nino).fullName
-        }
-        val amountPaidHeadingMessage: String = beneficiaryFullNameOpt match {
-          case Some(beneficiaryName) => Message("datePaid.event2.checkYourAnswersLabel", beneficiaryName)
-          //TODO: what to do when no name is pulled back here
-          case _ => Message("datePaid.event2.checkYourAnswersLabel", "")
-        }
+        val beneficiaryName = answers.get(MembersDetailsPage(Event2, index, Event2MemberPageNumbers.SECOND_PAGE_BENEFICIARY))
+          .map { g => MembersDetails(g.firstName, g.lastName, g.nino).fullName }
```

Comment:
Is there a more appropriate naming for `g`?

Reply:
Fixed - changed name